addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -564,6 +564,7 @@ X2ManyFieldDialog.props = {
     save: Function,
     title: String,
     delete: { optional: true },
+    button_text: String,
```

Comment:
Multiple things here:
- Don't use snake_case in js, we use camelCase.
- Can't we name it something more obvious? Maybe deleteButtonLabel ?
- I would make it optional and use the current value if not provided (aka "Remove") in the template.
- Not sure if we can add props in stable, definitely not required ones, but we will ask JS team to confirm after my review.

Reply:
Hello @awa-odoo

Reply:
Hello.

I mean that as the prop is optional, you should check it in the xml template and do "t-if prop" -> print the prop label, "t-else", print the existing label (so "Remove").

Even if it's always specified in your use case, this component could be used in other contexts where the prop is not specified (as it's optional) and we will need a label for the button in that case.

Does that make sense?

Cheers!

Reply:
Hello @awa-odoo 

Changes are done as per your suggestions can you please review it again :)